backend/src/main/java/com/bakdata/conquery/sql/conversion/cqelement/concept/filter/NumberFilterConverter.java

```diff
@@ -53,4 +57,25 @@ public Set<ConceptCteStep> requiredSteps() {
 		return (Class<? extends NumberFilter<IRange<? extends Number, ?>>>) CLASS;
 	}
 
+	/**
+	 * If there is a long range filter on a column of type MONEY, the filter value will represent a decimal with the point moved right 2 places right.
+	 * <p>
+	 * For example, the filter value {@code {min: 1000€, max: 2000€}} will be converted to {@code {min: 10,00€, max: 20,00€}}
```

Comment:
Hey Jonas, der Kommentar macht durch die Einheiten den Eindruck als würde hier absichtlich etwas falsch gemacht werden.
```suggestion
	 * For example, the filter value {@code {min: 1000ct, max: 2000ct}} will be converted to {@code {min: 10,00€, max: 20,00€}}
```

Reply:
Danke für's genau Lesen, so ergibt das natürlich Sinn 😄